experimental/PropertyDDS/packages/property-inspector-table/src/InspectorTableTypes.ts

```diff
@@ -22,12 +91,10 @@ export interface IColumns {
 /**
  * The interface for an entry of the visualization data array.
  */
-export interface IInspectorRow {
+export interface IInspectorRow extends IRowData<BaseProxifiedProperty>{
```

Comment:
Not for review, just a question. Is this another one type we should relax to decouple table from `propertyDDS`?

Reply:
IRowData should be the one, but probably we should reiterate on it.